modules/ide-integration/src/main/resources/pga/airavata-php-gateway/app/views/partials/gateway-preferences-block.blade.php

```diff
@@ -83,6 +83,7 @@ class="glyphicon glyphicon-plus"></span> Add a Compute Resource Preference
                         <div class="panel-group" id="cr-{{$accName}}-{{$indexGP}}">
                             @foreach( (array)$gp->profile->computeResourcePreferences as $indexCRP
                             => $crp )
+                            @if($crp->crDetails->enabled)
```

Comment:
I don't think we should hide compute preferences where the resource is disabled. Instead I think we should:
- display a message that the resource is disabled
- display the form but make it readonly

Reply:
@machristie Sure, I will unhide the deleted resources and show them as readonly with a message that resource is disabled. I might take a while to do this change as I am short of time right now. 

The reason I hid them was because anything related to compute resource preference is of no use after the compute resources are deleted. It will add extra information on the screen which is of no use to the user.